plugins/serializers/graphite/graphite.go

```diff
@@ -28,6 +28,9 @@ var (
 	)
 
 	fieldDeleter = strings.NewReplacer(".FIELDNAME", "", "FIELDNAME.", "")
+
+	allowedCharsTagName  = regexp.MustCompile(`[^ "-:\<>-\]_a-~\p{L}]`)
```

Comment:
Should we rename the other allowedChars on line 19 to strictAllowedChars to distinguish the two

Reply:
Named each variable for what mode they apply to.